rllib/algorithms/algorithm_config.py

```diff
@@ -2409,6 +2411,17 @@ def multi_agent(
         of IDs is properly converted into a dict mapping these IDs to PolicySpecs.
 
         Args:
+            enable_multi_agent: Whether multi-agent mode should be manually enabled
```

Comment:
In which use case does this play a role? Are there users with environments that are multi-agent, but can be played single-agent?

Reply:
I guess this is the case, if we train only a single policy and there are jultiple agents in the environment using all this single policy. Got you.